docs_src/microservices/device/services/device-mqtt/ProtocolProperties.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Device MQTT - Protocol Properties
+---
+
+# Device MQTT - Protocol Properties
+
+This service defines the following Protocol Properties for each defined device that supports 2-way communications.  These properties reside under the `mqtt` key in the `protocols` section of each device definition.
+
+| Property     | Description                                   |
+|--------------|-----------------------------------------------|
+| CommandTopic | MQTT topic for sending commands to the device |
+
```

Comment:
Add note about not require if only sending async data (one-way) Same as on REST here. https://docs.edgexfoundry.org/3.1/microservices/device/services/device-rest/ProtocolProperties/